rust/tw_bitcoin/src/tests/p2tr_script_path.rs

```diff
@@ -0,0 +1,138 @@
+use crate::{
+    brc20::{BRC20DeployInscription, BRC20MintInscription},
+    keypair_from_wif, Recipient, TXOutputP2TRScriptPath, TransactionBuilder, TxInputP2PKH,
+    TxInputP2TRScriptPath, TxOutputP2PKH,
+};
+use bitcoin::{PubkeyHash, Txid};
+use std::str::FromStr;
+use tw_encoding::hex;
+
+#[test]
+#[ignore]
+// WIP...
+fn mainnet_brc20_deploy() {
+    const EXPECTED_RAW_DEPLOY: &str = "020000000121a9074b8d5c9f0c57528bbe503e96a06bb5530253afcde740357a0aac11a5c4000000006b48304502210094f38d41dd28f1916d06b49747276cf418ba002efa4ff151a4167ed070a7e90b022020c6edc13f4a9e1222ee9e09b99ca205139ccf5439a65abb34d62ce694bafdc4012103c934b8fa66a55b0a1517e8b443140c844b43682d75a4260a040cd8d301d11019ffffffff022602000000000000225120971bffe440f17f394e728dfd499d0d35fbab4fd17d0ca307eb4bc6909eaed6a6825f0000000000001976a9147542c8ab655580ef2a3d1d93153030c73467f8ba88ac00000000";
+
+    let ALICE_WIF: &str = env!("ALICE_WIF");
+    let ORDINALS_SATOSHIS: u64 = env!("ORDINALS_SATOSHIS").parse().unwrap();
+    let MINER_FEE: u64 = env!("MINER_FEE").parse().unwrap();
+
+    let ORIGIN_TXID: &str = env!("ORIGIN_TXID");
+    let DEPLOY_INPUT_AMOUNT: u64 = env!("DEPLOY_INPUT_AMOUNT").parse().unwrap();
+    let DEPLOY_RETURN_AMOUNT: u64 = DEPLOY_INPUT_AMOUNT - ORDINALS_SATOSHIS - MINER_FEE;
+
+    let alice = keypair_from_wif(ALICE_WIF).unwrap();
+
+    // ## FIRST TRANSACTION: Deploy the BRC20 token.
+
+    // Claim input.
+    let input = TxInputP2PKH::builder()
+        .txid(Txid::from_str(ORIGIN_TXID).unwrap())
+        .vout(0)
+        .recipient(&alice)
+        .satoshis(DEPLOY_INPUT_AMOUNT)
+        .build()
+        .unwrap();
+
+    // Prepare BRC20 Deploy output for Alice.
+    let ticker = "qqqq".to_string();
+    let max = 1000;
+    let limit = None;
+    let decimals = None;
+
+    let brc20_deploy =
+        BRC20DeployInscription::new((&alice).into(), ticker.clone(), max, limit, decimals).unwrap();
```

Comment:
Do we need to support BRC20 Deploy and Mint functionalities in Wallet Core?
cc @hewigovens

Reply:
I guess we do not, but once the underlying structure works those methods do not cost us anything.